src/types/biztechEvent.ts

```diff
@@ -0,0 +1,14 @@
+// type definition for a BiztechEvent
+export type BiztechEvent = {
```

Comment:
pls update this before merging

Reply:
I copied the file you sent over into mine and updated the fetch function. let me know if this works 👍

Reply:
jk